### PR TITLE
feat: allowlist stargate queries #ntrn-547

### PR DESCRIPTION
**Pull request by NeverHappened**
Implements https://p2pvalidator.atlassian.net/browse/NTRN-547

Integration tests: https://github.com/neutron-org/neutron-integration-tests/pull/175
New contract for for new integration tests: https://github.com/neutron-org/neutron-dev-contracts/pull/21

### CODE REVIEW

File: wasmbinding/stargate_allowlist.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+		// interchaintxs
+		"/neutron.interchaintxs.Query/Params": &interchaintxstypes.QueryParamsResponse{},
+
+		// interchainqueries
+		"/neutron.interchainqueries.Query/Params": &interchainqueriestypes.QueryParamsResponse{},
+
+		// feeburner
+		"/neutron.feeburner.Query/Params":                    &feeburnertypes.QueryParamsResponse{},
+		"/neutron.feeburner.Query/TotalBurnedNeutronsAmount": &feeburnertypes.QueryTotalBurnedNeutronsAmountResponse{},
```
Comment (mmikev) [inline wasmbinding/stargate_allowlist.go]: We have these queries in our bindings

**Review by foxpy — COMMENTED** [review]
Looks decent, but I am hesitant to approve without any tests.

**Comment by NeverHappened** [comment]
Integration tests run (see if passed): https://github.com/neutron-org/neutron-tests/actions/runs/5521339064

**Comment by NeverHappened** [comment]
> Looks decent, but I am hesitant to approve without any tests.

I added tests just now ! :)

**Review by pr0n00gler — APPROVED** [review]

**Review by swelf19 — APPROVED** [review]

**Pull request merged**